commerce-theme-minium/commerce-theme-minium/src/css/minium/_variables.scss

```diff
@@ -31,12 +31,13 @@ $placeholder-colors: (
 	#639DFF
 );
 
-$minium-shadow: 0 10px 20px $color-shadow;
-$minium-shadow-focus: 0 3px 20px 0 rgba($color-theme, .5);
+$shadow: 0 10px 20px $color-shadow;
+$shadow-focus: 0 3px 20px 0 rgba($color-theme, .5);
 
-$minium-gutter: 20px;
-$minium-small-gutter: $minium-gutter / 2;
-$minium-big-gutter: $minium-gutter * 2;
+$xs-gutter: 5px;
+$sm-gutter: 10px;
+$gutter: 20px;
+$lg-gutter: 40px;
```

Comment:
variables should be sorted alphabetically within a "chunk" at least. But I'm fine maintaining the chunking organization rather than doing a pure alpha sort

Reply:
Hey @ethanbustad I would keep this as is because it follows a logic order

Reply:
Hmmm..... alright, that's fine I suppose.